src/core/main.cpp

```diff
@@ -191,6 +191,7 @@ int main( int argc, char * * argv )
 	"-x, --oversampling <value>	specify oversampling\n"
 	"				possible values: 1, 2, 4, 8\n"
 	"				default: 2\n"
+	"-32, --float			32bit float bit depth\n"
```

Comment:
@curlymorphic, IIRC a two character command line parameter `32` breaks the single dash rule.  For most CLIs this would assume you meant to supply `-3 -2`.

`-a` isn't used, which could signify the signals amplitude bit depth (I don't really like "a" but `-b` and `-d` are already taken unfortunately)

-Tres